inspirehep/modules/records/tasks.py

```diff
@@ -26,164 +26,23 @@
 
 from celery import shared_task
 from celery.utils.log import get_task_logger
-from elasticsearch.helpers import bulk, scan
+from elasticsearch.helpers import bulk
```

Comment:
can we just delete the file?

Reply:
It will require changing quite few places in the code...